target/linux/ath79/dts/qca9533_tplink_cpe210.dtsi

```diff
@@ -4,20 +4,9 @@
 #include <dt-bindings/gpio/gpio.h>
 #include <dt-bindings/input/input.h>
 
-#include "qca953x.dtsi"
+#include "qca9533_tplink_cpexxx.dtsi"
 
 / {
-	chosen {
-		bootargs = "console=ttyS0,115200n8";
-	};
-
-	aliases {
-		led-boot = &system;
-		led-failsafe = &system;
-		led-running = &system;
-		led-upgrade = &system;
-		label-mac-device = &eth0;
-	};
 
 	leds {
```

Comment:
I don't think this will work, as leds is already defined in parent DTSI. You will need to add a label to parent leds
`leds: leds {`
and then use `&leds` outside of the root node to add additional leds in this file (as you do for eth0 etc.).

Reply:
This compiles cleanly and works when tested on my CPE210 V2 and CPE220 V3